doc/design/parameter_average.md

```diff
@@ -0,0 +1,70 @@
+# Averaging Parameter Updates in PaddlePaddle
+
+## Why Averaging
+In large scale machine learning, it could take us a large number of iterations over the training data to achieve optimal values of parameters of our model. The size of training data could be very large for large scale training hence it is desirable if we can obtain the optimal parameters by going through the data in as few passes as we can.
+Polyak and Juditsky (1992) showed that asymptotically the test performance of simple average of parameters obtained by Stochastic Gradient Descent (SGD) is as good as that of parameters which minimize the empirical cost.
```

Comment:
It seems that this paragraph comes from the abstract of [this paper](https://arxiv.org/pdf/1107.2490.pdf) by Wei Xu. I think we need to add a citation here or rephrase this paragraph.

Reply:
Done.